apps/web/next-i18next.config.js

```diff
@@ -4,7 +4,7 @@ const { resolve } = require('path');
 module.exports = {
   i18n: {
     defaultLocale: 'en',
-    locales: ['en'],
+    locales: ['en', 'zht'],
```

Comment:
It seems like not all projects have the same locale settings. What's the reasoning behind this

Reply:
will double check if the i18n keys match with the json files under the `en` directory on `web` workspace and update the locale settings accordingly!